microceph/database/client_config_extras.go

```diff
@@ -0,0 +1,220 @@
+package database
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"github.com/canonical/lxd/shared/logger"
+	"github.com/canonical/microceph/microceph/api/types"
+	"github.com/canonical/microceph/microceph/common"
+	"github.com/canonical/microcluster/cluster"
+	"github.com/canonical/microcluster/state"
+)
+
+// Slice of ClientConfigItem(s)
+type ClientConfigItems []ClientConfigItem
+
+func (cci ClientConfigItems) GetClientConfigSlice() types.ClientConfigs {
```

Comment:
This is essentially converting between types.ClientConfigs and  ClientConfigItems right? Could possibly add some commentary around that